code/_onclick/pai.dm

```diff
@@ -1,5 +1,37 @@
 /mob/living/silicon/pai/ClickOn(var/atom/A, params)
```

Comment:
why are you no longer calling the parent? this is usually a sign that there's bad code here

Reply:
In order to get pAI's to have a functioning camera I had to create the file and add ``if(aicamera.in_camera_mode)`` onward. I didn't realize that creating an _onlick/pai.dm would cause them to lose their hotkeys as well. It didn't seem like using _onclick/cyborg.dm would work properly and there's no _onclick/silicon.dm. Just AI.dm and cyborg.dm, it looks like pai was falling into other_mobs.dm or something.